File: C++/TD5/Personne/personne.cpp
#include "personne.h"



Personne::Personne(string nom, string adresse) {
	m_nom = nom;
	m_adresse = adresse;
}

string Personne::getNom() {
	return m_nom;
}

string Personne::getAdresse() {
	return m_adresse;
}

void Personne::setNom(string nNom) {
	m_nom = nNom;
}

void Personne::setAdresse(string nAdresse) {
	m_adresse = nAdresse;
}

void Personne::afficher() {
	cout << "Nom : " << m_nom << endl << "Adresse : " << m_adresse << endl;
}

File: Java/TD2/src/Personne.java
abstract class Personne {
    String m_nom = "Henri Dupont";
    String m_adresse = "2 rue de la Paix, Paris";

    public Personne() {}

    public Personne(String m_nom, String m_adresse) {
        this.m_nom = m_nom;
        this.m_adresse = m_adresse;
    }

    public String getNom() {
        return m_nom;
    }

    public String getAdresse() {
        return m_adresse;
    }

    public void setNom(String m_nom) {
        this.m_nom = m_nom;
    }

    public void setAdresse(String m_adresse) {
        this.m_adresse = m_adresse;
    }

    public abstract String afficher();
}
